components/mdx/Spoiler.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import styles from '../../scss/spoiler.module.scss'
+type Props = {
+  name?: string
+}
+const Spoiler: React.FC<Props> = ({ children, name }) => {
+  const [status, setStatus] = React.useState({
```

Comment:
Just to keep the same standard between all React code.

```suggestion
import React, { useState } from 'react'
import styles from '../../scss/spoiler.module.scss'
type Props = {
  name?: string
}
const Spoiler: React.FC<Props> = ({ children, name }) => {
  const [status, setStatus] = useState({
```